go/teams/chain_parse.go

```diff
@@ -37,6 +39,19 @@ type SCTeamMembers struct {
 	None    *[]SCTeamMember `json:"none,omitempty"`
 }
 
+type SCTeamInvites struct {
```

Comment:
This is what's the server wants. It's done this way to be consistent with the `members`